_data/other.yml

```
@@ -115,6 +115,12 @@ websites:
       - totp
     doc: https://www.contentful.com/faq/two-factor-authentication/
 
+  - name: Craigslist
+    url: https://craigslist.org/
+    img: craigslist.png
+    twitter: craigslist
+    facebook: Craigslist-74561112489
```

```suggestion
    facebook: officialcraigslist
```
That page doesn't 404 for me and it looks nothing less official than the twitter account with it's last activity in 2009.
I really wonder if there is any point in adding those social media channels as they don't seem toncare about them at all.

---

I don't mind changing, but the page I linked to has 451,611 likes, while this one has 1,140. Let me know.

---

You can also try this URL (which is the one Facebook's HTML marks as canonical): https://www.facebook.com/pages/category/Advertising-Marketing/Craigslist-74561112489/

---

OK, I give up :( Let's go with the Facebook page that has 1% the number of likes.